### PR TITLE
Rename Gradle Enterprise to Develocity

### PR DESCRIPTION
**Pull request by welandaz**

### CODE REVIEW

**Review by alextu — APPROVED** [review]
LGTM, once merged smoke tests will need to be fixed to point to the new templates.

File: README.md
Comment (guylabs) [inline README.md]:
```suggestion
The `.injectDevelocityForGradle` creates an init script with the instrumentation logic and exports the path as `$DEVELOCITY_INIT_SCRIPT_PATH` environment variable.
```

**Review by guylabs — APPROVED** [review]
Please ensure that all build agents and CCUD's are updated to the latest version.

File: README.md
Comment (welandaz) [reply README.md]: Done

**Pull request merged**
